If-risk/SystemTime.cs

```diff
@@ -0,0 +1,22 @@
+namespace If_risk;
+/// <summary>
+/// Used for getting DateTime.Now(), time is changeable for unit testing
+/// </summary>
+public static class SystemTime
```

Comment:
vajag šitādu klasi?

Reply:
IInterface prasība, ka validFrom datums nevar būt pagātnē.
Lai tagadējo datumu varētu pārbaudīt testos to vajag kaut kā nofiksēt un es atradu šādu veidu.
Bez tās klases pēc, piemēram, 1 gada vairāki testi salūztu un būtu jāatjauno.